javaslang/src/test/java/javaslang/collection/LinkedHashMapTest.java

```diff
@@ -69,8 +69,14 @@ protected String className() {
 
     @Test
     public void shouldKeepOrder() {
-        CharSeq actual = LinkedHashMap.<Integer, Character> empty().put(3, 'a').put(2, 'b').put(1, 'c').foldLeft(CharSeq.empty(), (s, t) -> s.append(t._2));
-        assertThat(actual).isEqualTo(CharSeq.of("abc"));
+        List<Character> actual = LinkedHashMap.<Integer, Character> empty().put(3, 'a').put(2, 'b').put(1, 'c').foldLeft(List.empty(), (s, t) -> s.append(t._2));
+        Assertions.assertThat(actual).isEqualTo(List.of('a', 'b', 'c'));
```

Comment:
Yes - also IntMap and IntMultimap hide methods overridden by Map/Multimap because IntMap only extends Traversable. It is not sufficient that they are backed by Map/Multimap. This is the reason we did not recognize a bug...

With 3.0.0 we will change the collection type hierarchy 'a bit'

* Iterator will not be Iterable/Traversable anymore, instead it will be TraversableOnce
* Map/Multimap will extend Traversable&lt;V&gt; instead of Traversable&lt;K, V&gt;

This will make IntMap/IntMultimap superfluous. But for the overridden assert* methods I have currently no better solution...